src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformerTest.php

```diff
@@ -352,11 +362,13 @@ public function testReverseTransformWrapsIntlErrorsWithExceptions()
             $this->markTestSkipped('intl extension is not loaded');
         }
 
-        $this->iniSet('intl.use_exceptions', 1);
+        $initialUseExceptions = ini_set('intl.use_exceptions', 1);
 
         $this->expectException(TransformationFailedException::class);
         $transformer = new DateTimeToLocalizedStringTransformer();
         $transformer->reverseTransform('12345');
+
+        ini_set('intl.use_exceptions', $initialUseExceptions);
```

Comment:
Make sure that the ini value has to be reset even if the test fails. You can achieve that by wrapping the code into a `try`/`finally` block. You can also consider moving the cleanup part to the `tearDown()` method.

```suggestion
        $initialUseExceptions = ini_set('intl.use_exceptions', 1);

        $this->expectException(TransformationFailedException::class);
        
        try {
            $transformer = new DateTimeToLocalizedStringTransformer();
            $transformer->reverseTransform('12345');
        } finally {
            ini_set('intl.use_exceptions', $initialUseExceptions);
        }
```

Reply:
You're right. I wrapped all calls to `ini_set()` with `try/finally`